source/common/statistic_impl.cc

```diff
@@ -323,6 +323,14 @@ void SinkableHdrStatistic::recordValue(uint64_t value) {
   scope_.deliverHistogramToSinks(*this, value);
 }
 
+std::string SinkableHdrStatistic::tagExtractedName() const {
+  if (worker_id().has_value()) {
+    return fmt::format("{}.", worker_id().value()) + id();
```

Comment:
This seems weird.  Why use fmt::Format if you're still going to concatenate the worker_id with the id?

Couldn't you do either:
```worker_id().value()) + id() + "."```
or 
or use fmt::format to replace bother worker_id().value() and id() as their own separate interpolations?

Reply:
Thanks Nate. Changed to 
`fmt::format("{}.{}", worker_id().value(), id())`